config/.eslintrc.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
[nit] Probably, we don't need disabling eslint here because we have the `@openedx/frontend-build` in our dependencies.

Reply:
If we remove the linter exception, we get the following error
` 1:26  error  '@openedx/frontend-build' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies`